.github/workflows/ci.yml

```
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6' # Replace this with the minimum Julia version that your package supports.
+          - '1.10' # Replace this with the minimum Julia version that your package supports.
```

You could update setup-julia to v2 and use
```suggestion
          - 'min'
```